### ISSUE TITLE
Select Models

### ISSUE THREAD

**Issue by simplymathematics** [simplymathematics/621#2]
- [x] Use Log Likelihood, AIC, ROC curve,
- [ ] Evaluate Training Set
- [ ] Accuracy, Error, Precision, Sensitivity, Specificity, F1 score, AUC, conf matrix
- [ ] Make predictions with test set and interpret
